fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/functions/CompiledSqlFunction.java

```diff
@@ -92,12 +97,12 @@ public RecordMetaDataProto.PUserDefinedFunction toProto(@Nonnull final PlanSeria
 
     @Nonnull
     @Override
-    public RelationalExpression encapsulate(@Nonnull final List<? extends Typed> arguments) {
+    public Reference encapsulate(@Nonnull final List<? extends Typed> arguments) {
```

Comment:
Wouldn't you only ever put exactly one expression into the `Reference`? If so, why put it in a reference? Why not just leave it as it was?